From 0e535fa024870259a89875fe3ceb6a6a7af39032 Mon Sep 17 00:00:00 2001
From: peijiankang <peijiankang@kylinos.cn>
Date: Fri, 24 Jun 2022 14:36:45 +0800
Subject: [PATCH] Modified kylin-weather display is incomplete

---
 src/citycollectionwidget.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/citycollectionwidget.ui b/src/citycollectionwidget.ui
index ad0ec8c..6f98797 100644
--- a/src/citycollectionwidget.ui
+++ b/src/citycollectionwidget.ui
@@ -31,7 +31,7 @@
     <rect>
      <x>46</x>
      <y>24</y>
-     <width>56</width>
+     <width>121</width>
      <height>17</height>
     </rect>
    </property>
-- 
2.33.0

