From 020c2d7b007d457fa7d728eea9dde2eabc385681 Mon Sep 17 00:00:00 2001
From: tanyulong2021 <tanyulong@kylinos.cn>
Date: Wed, 13 Jul 2022 15:13:26 +0800
Subject: [PATCH] update translation file

---
 src/informationwidget.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/informationwidget.cpp b/src/informationwidget.cpp
index 7d4578b..d83eb3c 100644
--- a/src/informationwidget.cpp
+++ b/src/informationwidget.cpp
@@ -230,7 +230,7 @@ void Information::onSetForecastWeather(ForecastWeather forecastweather)
 
     //获取星期
     QDateTime current_date_time = QDateTime::fromString(forecastweather.dateTime,"yyyy-MM-dd");//将从服务器上获取到的日期处理成特定格式
-    QString current_week_1 = "今天";//current_date_time.toString("ddd");
+    QString current_week_1 = tr("Today");//current_date_time.toString("ddd");
     QString current_week_2 = current_date_time.addDays(+1).toString("ddd");
     QString current_week_3 = current_date_time.addDays(+2).toString("ddd");
     QString current_week_4 = current_date_time.addDays(+3).toString("ddd");
-- 
2.33.0

