diff -u -r indicator-china-weather-3.1.3/src/menumodule.cpp indicator-china-weather-3.1.3/src/menumodule.cpp
--- indicator-china-weather-3.1.3/src/menumodule.cpp	2021-11-04 16:29:04.000000000 +0800
+++ indicator-china-weather-3.1.3/src/menumodule.cpp	2023-06-20 16:03:20.000000000 +0800
@@ -20,7 +20,7 @@
 #include "menumodule.h"
 #include "xatom-helper.h"
 
-menuModule::menuModule(QWidget *parent = nullptr) : QWidget(parent)
+menuModule::menuModule(QWidget *parent) : QWidget(parent)
 {
     init();
 }
